src/components/Transcript/TranscriptSection.ts

```diff
@@ -102,7 +102,7 @@ export class TranscriptSection implements RedomComponent {
 
     const keywordClassNames = extractKeywordsClassNames(
```

Comment:
Z nějakého důvodu se to nevyrenderuje, u toho speakera je jen třída v tvaru "skw-", to se teda dělo už předtím, nevím, jestli tam není nějaka asynchronicita, která to takhle rozbíjí...